airflow-ctl-tests/pyproject.toml

```diff
@@ -0,0 +1,65 @@
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[build-system]
+requires = [ "hatchling==1.27.0" ]
+build-backend = "hatchling.build"
+
+[project]
+name = "apache-airflow-ctl-tests"
+description = "Airflow CTL tests for Apache Airflow"
+classifiers = [
+    "Private :: Do Not Upload",
+]
+requires-python = ">=3.10,!=3.13"
+authors = [
+    { name = "Apache Software Foundation", email = "dev@airflow.apache.org" },
+]
+maintainers = [
+    { name = "Apache Software Foundation", email="dev@airflow.apache.org" },
+]
+version = "0.0.1"
+
+dependencies = [
+    "apache-airflow-ctl",
+    "apache-airflow-devel-common",
+    "python-on-whales>=0.70.0",
```

Comment:
```suggestion
```

Reply:
Totally missed this :facepalm:  Thanks :)